ex/zuuli/src/electron/main.js

```diff
@@ -54,6 +54,19 @@ function createWindow() {
         console.log("OPEN", event, arg)
         shell.openExternal(arg)
     })
+    ipcMain.on('send', (event, id, json) => {
+        console.log("SEND", event, id, json)
+        // Zcash
+        warp.setActiveAccount(0, id)
+        const res = warp.sendMultiPayment(json)
```

Comment:
TODO: try to put into fork process? Even in the main process (not in the renderer), this is a lot of work and can cause BBOD in the Mac version.